modules/Hoot/config/domMetadata.js

```diff
@@ -313,30 +332,28 @@ export function exportDataForm( zipOutput ) {
             readonly: 'readonly',
             data: this.translations.map(t => t.NAME),
             value: 'OSM',
-            onChange: () => this.validate( exportComboId )
+            onChange: changeExport
         },
         {
             label: 'Export Format',
             id: exportFormatId,
             inputType: 'combobox',
             data: [ 'File Geodatabase', 'Shapefile', 'OpenStreetMap (OSM)', 'OpenStreetMap (PBF)' ],
             value: 'OpenStreetMap (OSM)',
-            onChange: () => {
-                const isFgdb = d3.select( `#${exportFormatId}` ).property( 'value' ) === 'File Geodatabase';
-
-
-                d3.select( `#${exportFgdbId}_container` )
-                    .classed( 'hidden', !isFgdb );
-
-                this.validate( exportFormatId );
-            }
+            onChange: changeExport
         },
         {
-            label: 'ESRI FGDB Template',
+            label: 'Append to ESRI FGDB Template?',
             id: exportFgdbId,
             class: 'hidden',
             inputType: 'checkbox',
-            value: 'append',
+            checked: false,
+            hidden: true
+        },
+        {
+            label: 'Include Hootenanny tags?',
+            id: exportHootTags,
+            inputType: 'checkbox',
```

Comment:
Makes the "Include Hootenanny tags" checkbox operable for OSM formatted exports.